templates/common/on-prem/files/keepalived-script-default-ingress.yaml

```
@@ -0,0 +1,6 @@
+mode: 0755
+path: "/etc/kubernetes/static-pod-resources/keepalived/scripts/chk_default_ingress.sh.tmpl"
+contents:
+  inline: |
+    #!/bin/bash
+    /host/bin/oc --kubeconfig /var/lib/kubelet/kubeconfig get ep -n openshift-ingress router-internal-default -o yaml  | grep 'ip:' | grep -q {{`{{.NonVirtualIP}}`}}
```

You have a unicode `\u00a0` NO-BREAK SPACE char in there. While I don't think it breaks anything I suggest you remove it:
```suggestion
    /host/bin/oc --kubeconfig /var/lib/kubelet/kubeconfig get ep -n openshift-ingress router-internal-default -o yaml | grep 'ip:' | grep -q {{`{{.NonVirtualIP}}`}}
```

See https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/openshift_machine-config-operator/2761/pull-ci-openshift-machine-config-operator-master-e2e-openstack/1438184066972127232/artifacts/e2e-openstack/gather-extra/artifacts/pods/openshift-openstack-infra_keepalived-qwzlrtts-bbcf1-45gx7-master-0_render-config-keepalived.log

---

Oh, nice catch. Thanks.
I'll fix that.